src/language-helper.ts

```diff
@@ -91,25 +91,35 @@ export class AppLocalization {
         this.resetLocalizedUI();
     }
 
+    // Fallbacks to a locale (e.g. en-gb to [en-gb, en, en-en] or just en to [en, en-en])
+    private variations(locale: string): string[] {
```

Comment:
Can we use `getNormalizedLanguageKeys` from `matrix-web-i18n` to remain consistent across the layers?